### DISCUSSION TITLE
Electricity Prices in the standard report

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#79] [General]
Hi，

Can I know more about the ReEDS electricity price? My understanding is that ReEDS has two types of electricity prices, one is the marginal electricity price and the other is the average electricity price. In the standard report, the electricity price is the marginal electricity price. Is this correct?

For the electricity price in the standard report, it includes state_rps, res_marg, oper_res, nat_gen, and load. 
state_rps refers to state level RPS requirements, nat_gen refers to national generation, load refers to electricity load, and oper_res refers to operating reserves. 

What does res_marg refer to?

Thanks！

**Comment by vcelloho** [comment]
In cases where the title uses price these are based upon the marginal prices on the modeled constraints. This can include energy, state rps, operating reserves, and planning reserve. The national average cost is based upon the annualized objective function as a fraction of energy produced.


res_marg refers to the resource adequacy or planning reserve. This is the value of firm capacity to the system.

**Reply by wanyingw193** [reply]
Thank you!

**Comment by wanyingw193** [comment]
Good afternoon, I have two questions related to the electricity price output from ReEDS.

(1) Is the output of the marginal electricity price a retail or wholesale price?

(2) Do i understand it correctly? The supply module will have the wholesale electricity price after each run and the regional and sector specific additions will be applied to get the retail electricity price which can be fed into the demand module.
